bitnami/mysql/values.yaml

```
@@ -193,7 +193,7 @@ primary:
   ## @param primary.existingConfiguration Name of existing ConfigMap with MySQL Primary configuration.
```

You need to update this comment too:
```suggestion
  ## @param primary.existingConfigmap Name of existing ConfigMap with MySQL Primary configuration.
```